src/index.html

```diff
@@ -5,8 +5,282 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet"
+          href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#Home">
+        <img class="header__logo"
+             src="images/logo.png"
+             alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__links is-active" href="#">apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__links" href="#">samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__links" href="#">смартфони і телефони
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__links"
+               href="#"
+               data-qa="nav-hover">ноутбуки і комп'ютери
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__links" href="#">гаджети
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__links" href="#">планшети</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__links" href="#">фото
+            </a>
```

Comment:
зкариваючий тег потрібно писати на рівні з відкриваючим, ящко ти не переносиш текст в тегу
```suggestion
            <a class="nav__links" href="#">фото</a>
```